tst/com/amazon/ion/benchmark/testFloatValidValue.isl

```diff
@@ -0,0 +1,9 @@
+schema_header::{
+}
+type::{
+    name: Float,
+    type: float,
+    valid_value: range:: [-1.447, 2.457],
```

Comment:
I am unsure how this test works. But Are we testing the generated data by validating it against the schema?
If yes, how does this test pass? Ion schema would not be able to understand `valid_value` constraint as it names as `valid_values`.

Reply:
> I am unsure how this test works. But Are we testing the generated data by validating it against the schema? Ion schema would not be able to understand `valid_value` constraint as it names as `valid_values` and would consider it open content.

Yes, the first step of testing is to load the ion schema file and then test the generated data by validating it against the schema. If the grammar isn't right it suppose to throw errors. I'll double check the test method and make edition in the follow commit.